Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -7906,6 +7906,13 @@ void HTMLMediaElement::markCaptionAndSubtitleTracksAsUnconfigured(ReconfigureMod
         scheduleConfigureTextTracks();
 }
 
+PlatformDynamicRangeLimit HTMLMediaElement::computePlayerDynamicRangeLimit() const
+{
+    if (!m_shouldSuppressHDR)
+        return m_platformDynamicRangeLimit;
+    return std::min(m_platformDynamicRangeLimit, PlatformDynamicRangeLimit::constrainedHigh());
```

Comment:
Don't we want `noLimit` by default?

Reply:
In the `m_shouldSuppressHDR==true` case, we do want to limit to max `constrainedHigh`, even for videos.
Or did I get my logic wrong?

Reply:
This is OK.